packages/react-native-worklets/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "react-native-worklets",
+  "version": "0.0.1",
+  "description": "🚀",
```

Comment:
This isn't a blocker for the merge, but we need to add a proper description here